source/modules/MEI.xml

```diff
@@ -1201,6 +1201,44 @@
       </valList>
     </content>
   </macroSpec>
+  <macroSpec ident="data.COURSENUMBER" module="MEI.stringtab" mode="add">
+    <desc xml:lang="en">In string tablature, the number of the course to be played, i.e., [1-9]+.</desc>
+    <content>
+      <rng:data type="positiveInteger" />
+    </content>
+  </macroSpec>
+  <macroSpec ident="data.COURSETUNING" module="MEI.stringtab">
```

Comment:
Same here: Since it is declared as belonging to `MEI.stringtab` module, it should be moved there.

Reply:
this should be connected to #1441

Reply:
Happy to take any suggestions! Patches welcome :-)